packages/web/docs/src/app/case-studies/case-studies-styles.css

```diff
@@ -0,0 +1,56 @@
+.case-studies {
+  --nextra-content-width: 992px;
+
+  /* hide leftmost column to left-align the case study */
+  & > div:first-of-type > :first-child {
+    @apply hidden;
+  }
+
+  & > div > article {
+    box-sizing: content-box;
+    max-width: 640px;
+  }
+
+  @apply text-green-1000 dark:text-white;
+
+  & article main > :is(h2, h3, h4, h5, h6, p, li) {
+    @apply text-green-1000 dark:text-white;
+  }
+
+  & article main > :is(h2, h3, h4, h5, h6) {
+    @apply border-none;
+  }
+
+  & img {
+    @apply rounded-2xl;
+  }
+
+  & div:has(> blockquote) {
```

Comment:
I'm considering refactoring it to a `Blockquote` component, but I'm kinda afraid that with this public API MDX will insert a paragraph there.

```mdx
<Blockquote>

  Sodales lobortis tristique ac metus commodo. 
  Amet etiam adipiscing arcu amet. Enim ornare cursus.
  Enim sagittis tincidunt aliquam eget lacinia curabitur. 
  Maecenas egestas dignissim enim ullamcorper auctor amet.

  <Blockquote.Cite 
    avatar="..."
    name="Name Surname"
    position="Position"  
  />
</Blockquote>
```